Infrastructure/EntityConfigurations/EmailLogConfigurations.cs

```diff
@@ -23,8 +23,8 @@ public void Configure(EntityTypeBuilder<EmailLog> builder)
             new EmailLog
             {
                 Id = 1,
-                CreatedDate = DateTime.UtcNow,
-                FinishProcessingDate = DateTime.UtcNow,
+                CreatedDate = new DateTime(2023, 6, 7, 20, 55, 43, 728, DateTimeKind.Utc).AddTicks(2661),
+                FinishProcessingDate = new DateTime(2023, 6, 7, 20, 55, 43, 728, DateTimeKind.Utc).AddTicks(2662),
```

Comment:
As you can see the previous config had `UtcNow` - the issue being that the previously created migration script then added in actual specific date/time as:
`new DateTime(2023, 6, 7, 20, 55, 43, 728, DateTimeKind.Utc).AddTicks(2661)`

Now in this PR when adding a new migration for the response status it was trying to again update the `CreatedDate` and `FinishProcessingDate` with the latest actual date when the migration was created.  So I had to hardcode this old config so that he migration creation doesn't believe there is a change.

Ideally the original configuration would have used an actual date/time (rather than `UtcNow`), but I hadn't spotted this at the time.